cmd/anonymizer/app/uiconv/reader.go

```diff
@@ -39,7 +39,7 @@ type Reader struct {
 func NewReader(capturedFile string, logger *zap.Logger) (*Reader, error) {
 	wd, err := os.Getwd()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("cannot get working directory: %w", err)
 	}
 	logger.Sugar().Infof("Current working dir is %s", wd)
```

Comment:
I think this is a debugging left-over, I don't see `wd` actually being used, if so it's best to delete L40-44